packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-platform-label.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import classNames from 'classnames';
+import { TModifiedTradingPlatformAvailableAccount } from 'Components/props.types';
+import { getPlatformLabel, getHeaderColor, platfromsHeaderLabel } from '../../Helpers/compare-accounts-config';
+
+type TCFDCompareAccountsPlatformLabelProps = {
+    trading_platforms: TModifiedTradingPlatformAvailableAccount;
+};
+
+const CFDCompareAccountsPlatformLabel: React.FC<TCFDCompareAccountsPlatformLabelProps> = ({ trading_platforms }) => {
```

Comment:
Don't use React.FC. We do not use it for our project.

Reply:
I have also commented on other PRs but they say that they are using the React.FC as in Accumulators PR.
Will try to remove it.